Bukkit/build.gradle

```diff
@@ -103,8 +103,10 @@ shadowJar {
     dependencies {
         include(dependency(":PlotSquared-Core"))
         include(dependency("io.papermc:paperlib:1.0.4"))
+        include(dependency("net.kyori:examination-api:1.0.0"))
+        include(dependency("net.kyori:examination-string:1.0.0"))
```

Comment:
We can safely remove the old `-SNAPSHOT` dependency located further down the file.
```gradle
include(dependency("net.kyori:examination-api:1.0.0-SNAPSHOT"))
include(dependency("net.kyori:examination-string:1.0.0-SNAPSHOT"))
````